File: Leetcode/javascript/number-of-good-pairs.js
// https://leetcode.com/problems/number-of-good-pairs/

var numIdenticalPairs = function(nums) {
    var goodPairs = 0
    for (i = 0; i < nums.length; i++) {
        for (j = 0; j < nums.length; j++) {
            if ((i < j) && (nums[i] == nums[j])) {
                goodPairs += 1
            }
        }
    }
    return goodPairs
};

File: Leetcode/python/number-of-good-pairs.py
# https://leetcode.com/problems/number-of-good-pairs/

def numIdenticalPairs(nums):
    # print(nums)
    goodPairs = 0
    for i in range(len(nums)):
        for j in range(len(nums)):
            if (i < j) and (nums[i] == nums[j]):
                # print(str(i) + ": " + str(nums[i]) + ", " + str(j) + ": " + str(nums[j]))
                goodPairs += 1
    return goodPairs